extensions/nuget/src/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { ActionPanel, List, Action, environment, Icon } from "@raycast/api";
+import { useEffect, useRef, useState } from "react";
+import PackageDetail from "./components/package-detail";
+import { searchNuget } from "./nuget-client";
+import { NugetPackage } from "./NugetPackage";
+import { GetCommandForCli, humanizeNumber, randomString } from "./utils";
+import { join } from "path";
+
+interface State {
+  isLoading: boolean;
+  query?: string;
+  items?: NugetPackage[];
+}
+
+export default function Command(): JSX.Element {
+  const [state, setState] = useState<State>({ isLoading: true });
+  const fetchId = useRef("");
+
+  useEffect(() => {
+    if (!state.isLoading) {
+      return;
+    }
+
+    const newId = randomString(10);
+
+    fetchId.current = newId;
+
+    searchNuget(state.query, newId).then((res) => {
+      const _fetchid = res.fetchId;
+      if (fetchId.current === _fetchid) {
+        setState((oldState) => ({ ...oldState, isLoading: false, items: res.data }));
+      }
+    });
+  }, [state]);
+
+  return (
+    <List
+      onSearchTextChange={(query: string) => {
+        // Perhaps query should be another useState??
+        setState((oldState) => ({ ...oldState, query: query, isLoading: true }));
+      }}
+      isLoading={state.isLoading}
+    >
+      {!state.isLoading &&
+        state.items &&
+        state.items.map((item) => {
+          return (
+            <List.Item
+              key={item.id}
+              icon={item.iconUrl || join(environment.assetsPath, "command-icon.png")}
+              title={item.title}
+              accessories={[
+                {
+                  text: humanizeNumber(item.totalDownloads),
+                  icon: Icon.Download,
+                },
+                {
+                  text: item.version,
+                  icon: Icon.Eye,
+                },
+              ]}
+              actions={
+                <ActionPanel>
+                  <Action.CopyToClipboard content={GetCommandForCli(item)} />
+                  <Action.Push title="Show Details" target={<PackageDetail package={item} />} icon={Icon.Eye} />
```

Comment:
Have you consider have "Show Details" as the main action and the copy one within the action panel?

Reply:
Move "Show Details" to first action